### PR TITLE
Data access scripts

### PR DESCRIPTION
**Pull request by wz7475**
Hi @prokotg,

after consultations with @szymczakpau  I investigated isue with data access:
- dvc is broken
- data on google drive is available, but can you make sure it's persistent and independ from your google account (please update link repo if necessary)
- I added script and enviroment specs to progamatically access data from drive

### CODE REVIEW

**Comment by prokotg** [comment]
If you believe DVC is broken, please file an issue providing reproducible steps. I was able to pull from DVC with a clean setup (clean repo pull, dvc install and dvc pull).

File: data_setup/.pdm-python
Comment (prokotg) [inline data_setup/.pdm-python:1]: That file should not be pushed

File: get_data.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# environment specified at ./data_setup
+python3 data_setup/download_data.py
+
+unzip unzip direct_download/models.zip
```
Comment (prokotg) [inline get_data.sh]: `unzip unzip`?

File: data_setup/pyproject.toml
Comment (prokotg) [inline data_setup/pyproject.toml:1]: I don't believe this package warrants a separate pyproject file, could we please merge required dependencies to setup?

File: data_setup/README.md
Comment (prokotg) [inline data_setup/README.md:1]: Duplicated from main readme

File: data_setup/download_data.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+import gdown
+
+link = "https://drive.google.com/drive/folders/1IwJjwSKpXYWWALKXM8nghqmlSvyu3BYq"
+gdown.download_folder(link, quiet=True)
```
Comment (prokotg) [inline data_setup/download_data.py]: Does it also fail quietly?

File: README.md [hunk 3]
```diff
@@ -5,16 +5,15 @@
 
 [![DOI](https://zenodo.org/badge/452699400.svg)](https://zenodo.org/badge/latestdoi/452699400)
 ### Data
+Data is hosted at google drive: https://drive.google.com/drive/folders/1IwJjwSKpXYWWALKXM8nghqmlSvyu3BYq
 
-Data is available via [DVC](https://dvc.org/) and can be obtained by running  following commands within repo root directory:
+You can either:
+- manually download and extract it
+- use script `get_data.sh`
+  - create dedicated environment
```
Comment (prokotg) [inline README.md]: if we merge packages required to download data into the hydramp package, we could get rid of this. Plus, there is only one package required for downloading , so dedicated environment might be a bit too much :))

**Comment by prokotg** [comment]
@wz7475 thanks for your contribution :)) Please have a look at the review, I'd like to merge this one first and then rebase #9 and review it as well.

**Review by prokotg — CHANGES_REQUESTED** [review]
@wz7475 thanks for your contribution :)) Please have a look at the review, I'd like to merge this one first and then rebase https://github.com/szczurek-lab/hydramp/pull/9 and review it as well.

**Comment by wz7475** [comment]
Forgot to attach log, I have to admit it is substantial part of PR - here it is:
```shell
git clone git@github.com:szczurek-lab/hydramp.git hydramp-clean
cd hydramp-clean

conda create python=3.8 -p ./env
conda activate ./env/

pip install dvc[gdrive]

dvc pull
Collecting                                                                                                                                          |9.83k [00:11,  867entry/s]
ERROR: failed to transfer '1edb29b3f644ef9409bf002c9a5a433d' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/1e/db29b3f644ef9409bf002c9a5a433d'
Fetching
ERROR: unexpected error - [Errno 2] No such file or directory: '/Users/wojtek/Documents/coding/hydramp-clean/.dvc/cache/1e/.04Yf0bZOMg3TNiM_UYnwgg.tmp'

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!


rm -rf .dvc/cache
dvc pull
Collecting                                                                                                                                           |0.00 [00:00,    ?entry/s]
ERROR: failed to transfer '67fb65fdac574914a3258ea0d96f0da2' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/67/fb65fdac574914a3258ea0d96f0da2'                                                                                                                                                                         
ERROR: failed to transfer '063da34cd18b39ec2bbb78c938091740' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/06/3da34cd18b39ec2bbb78c938091740'                                                                                                                                                                         
Fetching                                                                                                                                                                       
ERROR: unexpected error - [Errno 2] No such file or directory: '/Users/wojtek/Documents/coding/hydramp-clean/.dvc/cache/67/.yGxjQN33R_ty2ltRXKcFWw.tmp'                        
                                                                                                                                                                               
Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!  
```

I tried to succeed at `dvc pull` at:
- laptop with i9980h cpu with debian besed distro - ubuntu 23.04
- apple M1 laptop with mac os sonoma 14.4


in each case python/conda provided by [miniforge](https://github.com/conda-forge/miniforge)

File: data_setup/.pdm-python
Comment (wz7475) [reply data_setup/.pdm-python:1]: ok, removed

**Comment by wz7475** [comment]
> If you believe DVC is broken, please file an issue providing reproducible steps. I was able to pull from DVC with a clean setup (clean repo pull, dvc install and dvc pull).

Forgot to attach log, I have to admit it is substantial part of PR - here it is:
```shell
git clone git@github.com:szczurek-lab/hydramp.git hydramp-clean
cd hydramp-clean

conda create python=3.8 -p ./env
conda activate ./env/

pip install dvc[gdrive]

dvc pull
Collecting                                                                                                                                          |9.83k [00:11,  867entry/s]
ERROR: failed to transfer '1edb29b3f644ef9409bf002c9a5a433d' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/1e/db29b3f644ef9409bf002c9a5a433d'
Fetching
ERROR: unexpected error - [Errno 2] No such file or directory: '/Users/wojtek/Documents/coding/hydramp-clean/.dvc/cache/1e/.04Yf0bZOMg3TNiM_UYnwgg.tmp'

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!


rm -rf .dvc/cache
dvc pull
Collecting                                                                                                                                           |0.00 [00:00,    ?entry/s]
ERROR: failed to transfer '67fb65fdac574914a3258ea0d96f0da2' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/67/fb65fdac574914a3258ea0d96f0da2'                                                                                                                                                                         
ERROR: failed to transfer '063da34cd18b39ec2bbb78c938091740' - [Errno 2] No such file or directory: '1R3FBys0nEnGwMtfd-iH9C3OaBcgYS_2L/dvc_storage/06/3da34cd18b39ec2bbb78c938091740'                                                                                                                                                                         
Fetching                                                                                                                                                                       
ERROR: unexpected error - [Errno 2] No such file or directory: '/Users/wojtek/Documents/coding/hydramp-clean/.dvc/cache/67/.yGxjQN33R_ty2ltRXKcFWw.tmp'                        
                                                                                                                                                                               
Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!  
```

I tried to succeed at `dvc pull` using:
- laptop with i9980h cpu with debian besed distro - ubuntu 23.04
- apple M1 laptop with mac os sonoma 14.4


in each case python/conda provided by [miniforge](https://github.com/conda-forge/miniforge)

File: README.md (hunk 3 above)
Comment (wz7475) [reply README.md]: as requsted I moved `gdown` - lib for data download to `setup.py`

File: data_setup/download_data.py (hunk 2 above)
Comment (wz7475) [reply data_setup/download_data.py]: removed this flag, outputs error if occurs one (eg no network connection)

**Pull request merged**